### PR TITLE
feat(static): support authenticated Github access

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
#46

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -308,8 +308,9 @@ function staticBase(owner, repo, entry, ref, strain = 'default') {
  * @param root
  * @param esi
  * @param branch
+ * @param githubToken
  */
-function deliverPlain(owner, repo, ref, entry, root, esi = false, branch) {
+function deliverPlain(owner, repo, ref, entry, root, esi = false, branch, githubToken = undefined) {
```
Comment (tripodsan) [inline src/index.js]: I think the default for a missing arg is `undefined`...so no need to specify it here...

File: src/index.js [hunk 2]
```diff
@@ -321,6 +322,10 @@ function deliverPlain(owner, repo, ref, entry, root, esi = false, branch) {
     resolveWithFullResponse: true,
     encoding: null,
   };
+  if (githubToken) {
+    // eslint-disable-next-line dot-notation
+    rawopts.headers['Authorization'] = `token ${githubToken}`;
```
Comment (tripodsan) [inline src/index.js]: why ignoring the eslint style?

File: src/index.js (hunk 1 above)
Comment (tripodsan) [inline src/index.js]: I don't really like this very long list of arguments...why not pass an object? 
also see #19

Comment (stefan-guggisberg) [reply src/index.js]: I suggest we address this in a separate PR for #19.

**Pull request merged**